hypothesis-python/tests/cover/test_testdecorators.py

```diff
@@ -417,7 +417,7 @@ def foo(x):
     with raises(AssertionError):
         with capture_out() as out:
             foo()
-    assert failing == [2]
+    assert failing[0] <= 11
```

Comment:
This fixup is a bit scary, because it's so tightly coupled to the specifics of the continuation logic in a very non-obvious way.

It would be nice if there were some cleaner way to detect the number of multi-bug continuation test calls, or perhaps to turn off multi-bug continuation completely for some tests.

(Though the latter has the disadvantage of testing a less-realistic configuration.)

Reply:
Happy to take suggestions - I just didn't think the legibility of a few tests which already had magic numbers was worth adding more metadata to track.  Perhaps just a comment for each?

Reply:
I've commented and also strengthened this test.